src/Umbraco.Web.UI.Client/src/common/services/umbrequesthelper.service.js

```diff
@@ -15,16 +15,19 @@ function umbRequestHelper($http, $q, notificationsService, eventsService, formHe
          *
          * @description
          * This will convert a virtual path (i.e. ~/App_Plugins/Blah/Test.html ) to an absolute path
-         * 
+         *
          * @param {string} a virtual path, if this is already an absolute path it will just be returned, if this is a relative path an exception will be thrown
          */
         convertVirtualToAbsolutePath: function(virtualPath) {
```

Comment:
We can't really make this change as the method is no longer doing what it should be doing.
Before the change, this method would throw an exception given a relative path. It no longer does. Instead it just converts this into a (most likely) wrong/non-existing path `Umbraco.Sys.ServerVariables.application.applicationPath + relativePath`.